cdap-proto/src/test/java/co/cask/cdap/proto/security/CheckAuthorizedRequestTest.java

```diff
@@ -38,7 +39,7 @@ public void testValidation() {
     }
 
     try {
-      new CheckAuthorizedRequest(Ids.namespace("foo"), "bob", null);
+      new CheckAuthorizedRequest(Ids.namespace("foo"), bob, null);
       Assert.fail();
```

Comment:
Its helpful to know the reason or something about failure when look on bamboo failure (most importantly on the go) when you cannot look at the code readily. 
But its very trivial so its fine if you don't want to add it.